packages/framework-integration-tests/integration/provider-specific/azure/deployment/setup.ts

```diff
@@ -0,0 +1,21 @@
+import { deploy } from '../../../helper/cli-helper'
+import { sandboxPathFor } from '../../../helper/file-helper'
+import { overrideWithBoosterLocalDependencies } from '../../../helper/deps-helper'
+// Imported from another package to avoid duplication
+// It is OK-ish, since integration tests are always run in the context of the whole monorepo
+import { createSandboxProject } from '../../../../../cli/src/common/sandbox'
+import { setEnv } from '../../../helper/app-helper'
+import { AzureTestHelper } from '@boostercloud/framework-provider-azure-infrastructure'
+
+before(async () => {
+  await setEnv()
+  const configuredAssets = ['assets', 'assetFile.txt']
+  const sandboxedProject = createSandboxProject(sandboxPathFor('deploy'), configuredAssets)
+
+  await overrideWithBoosterLocalDependencies(sandboxedProject)
+
+  AzureTestHelper.ensureAzureConfiguration()
+  console.log('Deploying sandbox project...')
+  await deploy(sandboxedProject, 'azure')
+  console.log('...sleep finished. Let the tests begin 🔥!')
```

Comment:
```suggestion
  console.log('Finished! Let the tests begin 🔥!')
```

(as you removed the sleep 😉 )

Reply:
Right! thanks!